install_files/securedrop-grsec/DEBIAN/postinst

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# postinst script for securedrop-grsec
+#
+# see: dh_installdeb(1)
+set -e
+set -x
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+case "$1" in
+    configure)
+
+    # If the SecureDrop instance is running a rolled-back kernel, this will
+    # preserve the kernel boot priority specfied in its ordinal form by its
+    # canonical form. (In this specific case, 1>2 as GRUB_DEFAULT in
+    # /etc/default/grub indicates the instance is running 3.14.79-grsec)
+    # Otherwise, we want to run the latest 4.4 series kernel.
+    if grep -qE '^GRUB_DEFAULT=\"1>2\"' /etc/default/grub; then
+        sed -i 's/^\(GRUB_DEFAULT=\)"1>2"/\1\"Advanced options for Ubuntu>Ubuntu, with Linux 3.14.79-grsec\"/' /etc/default/grub
+    fi
+    # update grub to set the new default
+    update-grub2
```

Comment:
Whether we need to run this `update-grub` task here is dependent on the results of the grep above, so suggest moving it inside the if block.